PGIP/Server.hs

```diff
@@ -188,7 +194,9 @@ hetsServer opts1 = do
   putIfVerbose opts 1 $ "hets server is listening on port " ++ show port
   putIfVerbose opts 2 $ "for more information look into file: " ++ permFile
 #ifdef WARP3
-  runSettings (setPort port $ setTimeout 86400 defaultSettings) $ \ re respond -> do
+  runSettings (setOnExceptionResponse catchException $
+               setPort port $ 
```

Comment:
I tend to put the infix operators at the beginning of a next line (but that is matter of taste).
(Unfortunately in Scala the operator even must be at the end of a line to indicate its continuation)

Also I prefer to write "f . g $ h x" instead of "f $ g $ h x"